pkg/bootstrap/platform/gcp.go

```diff
@@ -128,9 +129,15 @@ func (e *gcpEnv) Metadata() map[string]string {
 	return md
 }
 
+var once sync.Once
+
 func parseGCPMetadata() (pid, npid, cluster, location string) {
 	gcpmd := gcpMetadataVar.Get()
-	log.Infof("Extract GCP metadata from env variable GCP_METADATA: %v", gcpmd)
+	if len(gcpmd) > 0 {
+		once.Do(func() {
```

Comment:
This seems pretty overkill.. why are we putting this on the log? Why not making it so we actually only run this code once? And fix https://github.com/istio/istio/issues/22011

Reply:
Good point. Revised. Thanks.